### DISCUSSION TITLE
Controller support

### DISCUSSION THREAD

**Discussion by BusinessClown445** [LostArtefacts/TRX#318] [Q&A] [answered]
Are there any plans to add controller support like in T2M?

**Comment by Soupdragon1** [comment]
Try renaming Tomb1Main.exe to TombATI.exe then launch the game via Steam then at least you'll get the Steam overlay and can use that to enable generic gampad support

**Comment by walkawayy** [comment]
Yes controller support is planned. See #279.

There's a config option in the Tomb1Main.json5 to enable Xbox controller support. I don't have one of those to test but that might work for you in the meantime.

**Comment by rr-** [comment, marked as answer]
Implemented in 2.9.
